### DISCUSSION TITLE
Ender3V2-422-BLTUBL-MPC-20231217 - BLTouch not probing bed

### DISCUSSION THREAD

**Discussion by GrandMasterGrinch** [mriscoc/Ender3V2S1#1218] [Q&A]
I installed Ender3V2-422-BLTUBL-MPC-20231217.bin on my Ender3 V2 printer and have BLTouch installed.  I'm not sure why it's not probing the bed when it's in Cura for G29 ; Probe Bed - BLTouch.  What it does is once bed temp and Nosel temp reaches it Homes Z and starts printing without probing the bed.  

G-Code Flovor in Cura is Marlin.

; Ender 3 Custom Start G-code
G92 E0 ; Reset Extruder
G28 ; Home all axes
G29 ; Probe Bed - BLTouch
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X0.1 Y20 Z0.3 F5000.0 ; Move to start position
G1 X0.1 Y200.0 Z0.3 F1500.0 E15 ; Draw the first line
G1 X0.4 Y200.0 Z0.3 F5000.0 ; Move to side a little
G1 X0.4 Y20 Z0.3 F1500.0 E30 ; Draw the second line
G92 E0 ; Reset Extruder
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X5 Y20 Z0.3 F5000.0 ; Move over to prevent blob squish

**Comment by mriscoc** [comment]
Check the Wiki for the nomenclature of the name and the type of leveling that the firmware has. Then go to "Slicer Scripts" in the wiki and choose the appropriate start script.

**Reply by GrandMasterGrinch** [reply]
I'm also noticing when I start a print.  The bed just sits for awhile, homes and it doesn't heat up the bed nor the nozzle.  Settings are in G-Code but printer failes to heat also.  If I were to manually set the temps, it will heat and then print.
